test/static-code

```diff
@@ -42,13 +42,22 @@ test_ruff() {
 }
 
 if [ "${WITH_PARTIAL_TREE:-0}" = 0 ]; then
+    test_mypy() {
+        command -v mypy >/dev/null || skip 'no mypy'
+        for pkg in systemd_ctypes ferny bei; do
+            test -e "src/cockpit/_vendor/${pkg}/__init__.py" || skip "no ${pkg}"
+        done
+        mypy --no-error-summary src/cockpit test/pytest
```

Comment:
Should we disable mypy caching iirc that gave some issues when running it separate, or is that solved or do we already do that in pyproject.toml

Reply:
The issue was being caused by an oddity in systemd_ctypes.  That's resolved now.  I don't see any reason to keep the cache disabled, since it saves quite a bit of time...